micrometer-observation-test/src/main/java/io/micrometer/observation/tck/ObservationRegistryCompatibilityKit.java

```diff
@@ -149,9 +149,7 @@ void checkedRunnableShouldBeObserved() throws Throwable {
         registry.observationConfig().observationHandler(handler);
         Observation observation = Observation.createNotStarted("myObservation", registry);
 
-        Observation.CheckedRunnable checkedRunnable = () -> assertThat(registry.getCurrentObservation())
```

Comment:
These variables were there explicitly because this way we won't test a separate scenario (overloaded method) accidentally and it might help to the reader of the test too.

Reply:
Thanks. This was actually calling the wrong overload. I've reverted that part of the change now.